.github/ISSUE_TEMPLATE/5-CUPERTINO_HTTP_REQUEST.md

```diff
@@ -0,0 +1,11 @@
+---
+name: `package:cupertinohttp` ⟶ I'd like a new feature.
```

Comment:
@brianquinlan – you're on changing all of the package names, right? Or waiting until later?

Reply:
I'd like to do that in a follow-up PR - there is already a lot here.